src/ratelimit/ratelimit.service.ts

```diff
@@ -46,7 +46,7 @@ export class RateLimitService {
      */
     public async shouldLimit(bucket: RateLimitBucket, key: string): Promise<boolean> {
         const baseCacheKey: string = RateLimitService.makeCacheKey(bucket, key);
-        const blockedCacheKey: string = `blocked${baseCacheKey}`;
+        const blockedCacheKey = `blocked${baseCacheKey}`;
```

Comment:
Removing the type declaration here.

Reply:
ESLint considers it redundant. I can find the rule at some point, but I think it's part of the `extends` rules.

Reply:
I like the consistency of declaring a type.  That being said, if we want to go the other, thats fine too